UD brakes the build

```
In file included from ../../../src/uct/ib/ud/ud_ep.c:10:0:
../../../src/uct/ib/ud/ud_inl.h: In function 'uct_ud_iface_get_next_pending':
In file included from ../../../src/uct/ib/ud/ud_verbs.c:25:0:
../../../src/uct/ib/ud/ud_inl.h: In function 'uct_ud_iface_get_next_pending':
../../../src/uct/ib/ud/ud_inl.h:17:18: error: variable 'ret' set but not used [-Werror=unused-but-set-variable]
     ucs_status_t ret;
                  ^
../../../src/uct/ib/ud/ud_inl.h:17:18: error: variable 'ret' set but not used [-Werror=unused-but-set-variable]
     ucs_status_t ret;
                  ^
```

--- a/src/uct/ib/ud/ud_inl.h
+++ b/src/uct/ib/ud/ud_inl.h
@@ -14,7 +14,6 @@ static inline ucs_status_t uct_ud_iface_get_next_pending(uct_ud_iface_t *iface, 
                                                          uct_ud_neth_t *neth)
 {
     uct_ud_ep_t *ep;
-    ucs_status_t ret;
     ucs_queue_elem_t *elem;
 
     if (!uct_ud_iface_can_tx(iface)) {
@@ -25,7 +24,6 @@ static inline ucs_status_t uct_ud_iface_get_next_pending(uct_ud_iface_t *iface, 
     elem = ucs_queue_pull_non_empty(&iface->tx.pending_ops);
     ep = ucs_container_of(elem, uct_ud_ep_t, tx.pending.queue);
     if (ep->tx.pending.ops & UCT_UD_EP_OP_ACK) {
-        ret   = UCT_UD_EP_OP_ACK;
         *r_ep = ep;
         uct_ud_neth_ctl_ack(ep, neth);
          --iface->tx.available;

